public/css/output.css

```diff
@@ -2328,6 +2328,66 @@ html {
   opacity: 1;
 }
 
+.file-input {
```

Comment:
I'm a little bit confused by this file. I've noticed this in other PRs that it seems to contain very big changes. Are you writing this code yourself? Or is it generated somehow?

If it's generated, it would be nice to be able to commit whatever file is responsible for generating this file. That's the file that you've actually changed, and as a reviewer I'm more concerned with that because that's the file that will need to be maintained by our development team from now until forever :)

Reply:
Ah cool. I've read up on this a bit now, and I think what you're doing is probably normal. I just noticed your `dev` script that looks like this too:
```
"dev": "concurrently \"nodemon app.js\" \"tailwindcss -i ./public/css/style.css -o ./public/css/output.css --watch\" ",
```

which is pretty nice! I probably wouldn't want to rely on that in production, just in case a developer decided to just run `nodemon app.js` and somehow never ran the `tailwindcss -i ./public/css/style.css -o ./public/css/output.css` part.

However, I think for this project since you are a single dev it's fine. What I would do in prod would be that this would always get run at build time:

`tailwindcss -i ./public/css/style.css -o ./public/css/output.css`

I'm not sure what your production setup looks like actually. We can chat about that at another time, but for now I think this is perfectly fine :)

> I'm using TailwindCSS. I think they are auto-generated based on the tags that I've used. I'm looking at the purge option to remove the unused style in production mode.